docs/development/best-practices.md

```diff
@@ -0,0 +1,133 @@
+# Best practices
+
```

Comment:
```suggestion

This document explains our best practices.
Follow these best practices when you're working on our code.

```

I'd like a short introduction to the contents of the page, and also explain why it's relevant/needed for the reader.

I think our goal here is to have the reader actually follow these best practices when they're working on our code?